docs/core/install/how-to-detect-installed-versions.md

```diff
@@ -18,13 +18,15 @@ If any command in this article fails, you don't have the runtime or SDK installe
 
 ## Check SDK versions
 
-You can see which versions of the .NET Core SDK are currently installed with a terminal. Open a terminal and run the `dotnet --list-sdks` command.
+You can see which versions of the .NET Core SDK are currently installed with a terminal. Open a terminal and run the following command:
+
+```dotnetcli
+dotnet --list-sdks
+```
```

Comment:
I prefer the way it was before. Can we just change the code language to dotnetcli and leave the command with the output? New version in preview:

![image](https://user-images.githubusercontent.com/24882762/71692221-29d19c00-2d5e-11ea-8620-c2dbf440df56.png)

Reply:
Yeah, these should be reverted. In this case, highlighting isn't that important, we shouldn't have two blocks next to each other. However, a sentence could be introduced that shows this is the output.

>When the command finishes, the list of installed .NET Core SDKs are displayed.

Something like that. It breaks it up and allows the coloring of the command.

Reply:
@Thraka, I've added a text between them.

Reply:
I think it looks fine with the added text now. Resolve this?